### ISSUE TITLE
关于损失函数的疑问

### ISSUE THREAD

**Issue by lzm-01** [alexhe101/MSDDN#3]
非常感谢你的工作！其次我有两个疑问：
1，`loss = self.loss(y, ms_image) / (self.cfg['data']['batch_size'] * 2)` 这里的×2没有理解
2，论文中的entire loss在代码中如何体现？

**Comment by alexhe101** [comment]
1. 乘2是框架代码里原有的，可以不用在意；
2. 开源的代码里没有提供fft loss 的实现，可以参考https://github.com/sunny2109/SAFMN/blob/main/basicsr/losses/losses.py#L258
